packages/react/src/components/TimePicker/TimePicker.tsx

```diff
@@ -7,10 +7,12 @@
 
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { usePrefix } from '../../internal/usePrefix';
 import deprecate from '../../prop-types/deprecate';
 import { ForwardRefReturn, ReactAttr } from '../../types/common';
+// @ts-ignore
+import { WarningFilled, WarningAltFilled } from '@carbon/icons-react';
```

Comment:
This error: Cannot find module '@carbon/icons-react' or its corresponding type declarations.ts(2307)
But the code works fine. I just added that to avoid the error
![Screenshot 2023-05-23 at 09 12 25](https://github.com/carbon-design-system/carbon/assets/52183462/db7edb7e-3575-4608-9ef0-27b0e338e22c)

Reply:
I removed, since other `.tsx` files don't ignore this error

Reply:
🤔 do we need to declare the icons package somewhere @tay1orjones ? or are we good to ignore this warning? 😅. Seeing it locally too

Reply:
Yeah there's no types for `@carbon/icons-react` yet https://github.com/carbon-design-system/carbon/issues/13143 https://github.com/carbon-design-system/carbon/issues/11317


https://ibm-studios.slack.com/archives/C03C8VASVED/p1684310836583939?thread_ts=1676663597.878789&cid=C03C8VASVED